pkg/docker/parseimage_test.go

```diff
@@ -35,38 +35,67 @@ func TestParseImageIDString(t *testing.T) {
 		shouldPass  bool
 	}{
 		{
-			description: "valid image name",
+			description: "valid format",
+			prefix:      "docker-pullable://",
+			name:        "abc",
+			sha:         "cb4983d8399a59bb5ee6e68b6177d878966a8fe41abe18a45c3b1d8809f1d043",
+			shouldPass:  true,
+		},
+		{
+			description: "valid format with 2 directories",
 			prefix:      "docker-pullable://",
 			name:        "abc/def",
 			sha:         "cb4983d8399a59bb5ee6e68b6177d878966a8fe41abe18a45c3b1d8809f1d043",
 			shouldPass:  true,
 		},
 		{
-			description: "no prefix",
+			description: "valid format with 3 directories",
```

Comment:
Is 3 directories somehow different than 2?

Reply:
No @rrati.

Reply:
Then why add a test for 3 dirs if 2 is the same?

Reply:
Removed 3 dirs test case